src/index.js

```diff
@@ -103,7 +104,7 @@ let program = yargs
     boolean: true
   })
   .strict(true)
-  .alias('help', 'h')
+  .alias('h', 'help')
```

Comment:
Could you please set the `scriptname`: https://github.com/yargs/yargs/blob/master/docs/api.md#scriptname0
I think we need it to be empty to preserve the current help output, though maybe some people would prefer it set to `mup`.

Reply:
Done.